enterprise-metrics/Makefile

```diff
@@ -12,8 +12,12 @@ help:
 	@awk 'BEGIN {FS = ": ##"; printf "Usage:\n  make <target>\n\nTargets:\n"} /^[a-zA-Z0-9_\.\-\/%]+: ##/ { printf "  %-45s %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
 
+vendor: ## Vendor the Jsonnet libraries.
+vendor: jsonnetfile.json
+	jb install
+
 test/vendor: ## Vendor the test Jsonnet libraries.
-test/vendor: test/jsonnetfile.json test/jsonnetfile.lock.json
+test/vendor: test/jsonnetfile.json
```

Comment:
I think we may not need this change. It's caused by the lock files not being committed due to a top-level `.gitignore` file. For consistency, I think we are best to commit those lock files and not regenerate them and then this dependency works again.

This will also mean that we don't need the `clean` target below.